CDL-Dryad/dryad-app
# Journal email -- submission system

For https://github.com/CDL-Dryad/dryad-product-roadmap/issues/1188

This builds on https://github.com/CDL-Dryad/dryad-app/pull/443, so you can either review that first, or just this one for everything together.

This PR connects the Manuscript model to the Dryad submission system, and removes reliance on the v1 journal module. 

To test:
1. Ensure the GMail connection is authorized and journal codes are imported, as described in https://github.com/CDL-Dryad/dryad-app/pull/443
2. Parse at least one email message, as described in https://github.com/CDL-Dryad/dryad-app/pull/443
3. Create a new Dryad dataset in the submission system.
4. Enter the journal name and manuscript number associated with the email you parsed, and click "Import Manuscript Metadata"
5. All of the details from the email should get imported into the dataset.
